### ISSUE TITLE
Passing null to parameter is deprecated - php 8

### ISSUE THREAD

**Issue by Luensche** [petitssuisses/piwigo-ShareAlbum#95]
When I open a shared album via link, I see the following error messages:

```
Deprecated
: strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated in
/config/www/plugins/ShareAlbum/main.inc.php
on line
262


Deprecated
: substr(): Passing null to parameter #1 ($string) of type string is deprecated in
/config/www/plugins/ShareAlbum/main.inc.php
on line
263


Deprecated
: substr(): Passing null to parameter #1 ($string) of type string is deprecated in
/config/www/plugins/ShareAlbum/main.inc.php
on line
266
```
[Piwigo](https://de.piwigo.org/) 13.8.0 
    Betriebssystem: Linux
    PHP: 8.1.20
    MySQL: 8.0.33 [2023-07-17 17:31:00]
    Grafikbibliothek: ImageMagick 7.1.0-62

**Comment by Luensche** [comment]
May be fixed by #96.

**Comment by petitssuisses** [comment]
Fixed with #96
